docker/user_demos.sh

```diff
@@ -22,7 +22,12 @@ if [ "$PYTHON" = "miniconda" ]; then
       # delete GPU deps
       sed -r -e '/^\s*- (astra-toolbox|tigre).*/d' -e '/^\s*- \S+.*#.*GPU/d' environment.yml > environment-sirf.yml
     fi
-    conda env update --file environment-sirf.yml
+    # do not install CIL from conda if SuperBuild already built it
+    if test "$(conda list '^cil$')" != "[]"; then
+      # delete CIL package from the environment file
+      sed -r -i -e '/^\s*- (cil).*/d' environment-sirf.yml
+    fi
+    conda env update --file environment-sirf.yml -v
```

Comment:
up to you if this is needed, idk.

```suggestion
    conda env update --file environment-sirf.yml
```